docs/network-restrictions.md

```diff
@@ -27,16 +27,35 @@ If that image won't be accessible from the node, a custom image can be specified
 -node-pause-image="192.168.1.1:5000/kubernetes/pause:3.1"
 ```
 
-For ContainerLinux nodes the [hyperkube](https://github.com/kubernetes/kubernetes/tree/master/cluster/images/hyperkube) image must be accessible as well.
-This is due to the usage of the [kubelet-wrapper](https://github.com/coreos/coreos-kubernetes/blob/master/Documentation/kubelet-wrapper.md).
 
-By default the image `k8s.gcr.io/hyperkube-amd64` will be used.
-If that image won't be accessible from the node, a custom image can be specified on the machine-controller:
+## Kubelet images
+
+### CoreOS ContainerLinux
```

Comment:
We should mention that CoreOS will not work with v1.19 and that we highly advise migrating to Flatcar.

Reply:
CoreOS does not work already